source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContainerView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ACRContainerView.swift
+//  AdaptiveCards
+//
+//  Created by uchauhan on 27/07/22.
+//
+
+import AdaptiveCards_bridge
+import AppKit
+
+class ACRContainerView: ACRContentStackView {
+    private (set) lazy var backgroundImageView: ACRBackgroundImageView = {
+        let view = ACRBackgroundImageView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    private lazy var widthConstraint = widthAnchor.constraint(equalToConstant: 30)
+    private lazy var backgroundImageViewBottomConstraint = backgroundImageView.bottomAnchor.constraint(equalTo: bottomAnchor)
+    private lazy var backgroundImageViewTopConstraint = backgroundImageView.topAnchor.constraint(equalTo: topAnchor)
+    private lazy var backgroundImageViewLeadingConstraint = backgroundImageView.leadingAnchor.constraint(equalTo: leadingAnchor)
+    private lazy var backgroundImageViewTrailingConstraint = backgroundImageView.trailingAnchor.constraint(equalTo: trailingAnchor)
+    
+    override func setupViews() {
+        addSubview(backgroundImageView)
+        super.setupViews()
+    }
+    
+    override func setupConstraints() {
+        super.setupConstraints()
+        backgroundImageViewTopConstraint.isActive = true
+        backgroundImageViewLeadingConstraint.isActive = true
+        backgroundImageViewTrailingConstraint.isActive = true
+        backgroundImageViewBottomConstraint.isActive = true
+    }
```

Comment:
If set constraint is going to be called every time
and you are not changing backgroundImageView constraints, might as well declare them here instead of creating lazy var

Reply:
maybe it's same. and this stuff is already created in ACRColumnView Class. we are create separate class for container view because we don't want to affect other property while changing. right now we do as much as less changes on old rendering flow.